libminifi/test/unit/GeneralUtilsTest.cpp

```diff
@@ -130,7 +130,7 @@ TEST_CASE("GeneralUtils::invoke FunctionObject", "[invoke function object]") {
   REQUIRE(false == utils::invoke(&free_function, false));
 
   const auto n = 3;
-  const auto int_timesn = [n](const int i) { return n * i; };
+  const auto int_timesn = [=](const int i) { return n * i; };
```

Comment:
I prefer the old version here.

Reply:
Removed the capture altogether as it is not needed for this test. Updated in 267cda83a77a5a6d05f9df714d0c923814351826